package.json

```diff
@@ -4,7 +4,7 @@
   "description": "amplify-category-api",
   "scripts": {
     "test-changed": "lerna run test --since main",
-    "test": "lerna run test",
+    "test": "lerna run test -- -u",
```

Comment:
Accidental commit?

Reply:
Oh yeah, just committed that since yarn test -- -u wasn't working from root. I'll revert (though I expect to see some changes required in E2E tests from the run, so will wait til those have succeeded/failed to get more data)

Reply:
Think it's just 'yarn test -u' without the additional dashes